packages/wonder-blocks-popover/components/popover-content-core.js

```diff
@@ -0,0 +1,125 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import type {AriaProps, StyleType} from "@khanacademy/wonder-blocks-core";
+import Colors from "@khanacademy/wonder-blocks-color";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {icons} from "@khanacademy/wonder-blocks-icon";
+import IconButton from "@khanacademy/wonder-blocks-icon-button";
+
+import type {Placement} from "@khanacademy/wonder-blocks-tooltip";
+
+type Props = {|
+    ...AriaProps,
+
+    /**
+     * The content to render inside the popover.
+     */
+    children: React.Node,
+
+    /**
+     * When true, the close button is shown; otherwise, the close button is not shown.
+     */
+    closeButtonVisible?: boolean,
+
+    /**
+     * Whether we should use the default light color scheme or switch to a
+     * different color scheme.
+     */
+    color: "blue" | "dark" | "light",
+
+    /**
+     * The placement of the popover window with respect to the anchor.
+     */
+    placement: Placement,
+
+    /**
+     * Called when the popover closes
+     */
+    onClose?: () => mixed,
+
+    style?: StyleType,
+|};
+
+/**
+ * This is the base popover container. It’s used internally by all the variants.
+ * Also, It can be used to create flexible popovers.
+ */
+export default class PopoverContentCore extends React.Component<Props> {
+    static defaultProps = {
+        color: "light",
+        placement: "bottom",
+        closeButtonVisible: true,
+        "aria-label": "Close Popover",
```

Comment:
This is the label for the close button not the whole popover content.  I wonder if we should have a separate prop that's "close-button" specific.

Reply:
Agreed, `aria-label` in this case is too open to interpretation, so it would be better to make this prop name more related to the close element.